docs/pages/foundations/animation/guidelines.js

```diff
@@ -0,0 +1,215 @@
+        />
+        <MainSection.Subsection
+          title="Axis considerations"
+          description="Animations along an axis can be used to show transitions and educate users about navigation patterns. For example, items could animate along the z-axis to help relay stacking or depth, while items animated along an x-axis or y-axis can help relay navigation patterns, like a Sheet that slides in and out from the side of the screen, or a toast that pops up from the bottom of the screen."
+        />
+        <MainSection.Subsection
+          title="Animation origin"
+          description={`
+          Keep transitions direct, meaning the animation starts from a logical location that gives the moving object the shortest distance to travel to its final destination. In other words, do not animate a Toast from the top of the screen down to the bottom of the screen if the final resting point is the bottom of the screen. Instead, begin the animation from the bottom of the screen.
+
+          Also consider collisions when choosing animation origins. If Notification cards can stack, they may animate in from the top right of a screen, but fade out to avoid colliding with incoming notifications.
+          `}
+        />
+        <MainSection.Subsection
+          title="Duration"
+          description="Animations should be direct and purposeful, meaning they only last as long as necessary to achieve the desired affect. For [accessibility reasons], avoid animations longer than 5 seconds. Most animations happen in a matter of milliseconds to avoid distracting the user. In most cases, aim to keep animations under one second. In the future, we will provide design tokens for desired durations in product."
+        />
+        <MainSection.Subsection
+          title="Easing and velocity"
+          description={`
+          The timing function of an animation should be determined by the property and object being animated.
+
+          Properties unrelated to motion (like color and opacity) will not likely benefit from easing and can be done linearly. For example, a background color of a button that expands or changes upon hover can happen linearly as it provides feedback to the user about their cursor position.
+
+          However, people experience motion non-linearly, so objects being moved can benefit from an easing timing function. For example, an object transitioning onto the screen from the right side may use ease-out as it slides into view to create visibility as fast as possible, and then ease-in upon its departure to create a smooth transition back to the main screen.
+
+          [Learn more about easing functions](https://www.smashingmagazine.com/2021/04/easing-functions-css-animations-transitions/#cubic-b%C3%A9zier-functions)
```

Comment:
The placement of this link is a bit awkward. Can we include it in the preceding paragraph?